modules/RTC/TraceablePeerConnection.js

```diff
@@ -26,6 +26,8 @@ const SIM_LAYER_1_RID = '1';
 const SIM_LAYER_2_RID = '2';
 const SIM_LAYER_3_RID = '3';
 const SIM_LAYER_RIDS = [ SIM_LAYER_1_RID, SIM_LAYER_2_RID, SIM_LAYER_3_RID ];
+const SIM_LAYER_BITRATES = [ 200000, 700000, 2500000 ];
```

Comment:
can you add a unit here? `SIM_LAYER_BITRATES_BPS`?

Reply:
ok sure